spec/example_app/db/seeds.rb

```diff
@@ -19,7 +19,7 @@
   )
 end
 
-product_attributes = YAML.load_file("./db/seeds/products.yml")
+product_attributes = YAML.load_file(File.expand_path('../db/seeds/products.yml', __dir__))
```

Comment:
Line is too long. [90/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.